From 727de4692d731655dea96702aa099f4f4d3a5203 Mon Sep 17 00:00:00 2001
From: Bogdan Hamciuc <bogdan.hamciuc@nxp.com>
Date: Mon, 21 Mar 2016 16:10:01 +0200
Subject: [PATCH 203/226] fsl-dpaa2: eth: Update description of DPNI counters

Update description of DPNI counters presented with "ethtool -S".

Signed-off-by: Bogdan Hamciuc <bogdan.hamciuc@nxp.com>
(cherry picked from commit f68aab60355d00af13fdff2ded7bf38809beacd3)
---
 drivers/staging/fsl-dpaa2/ethernet/dpaa2-ethtool.c |    9 ++++++---
 1 file changed, 6 insertions(+), 3 deletions(-)

--- a/drivers/staging/fsl-dpaa2/ethernet/dpaa2-ethtool.c
+++ b/drivers/staging/fsl-dpaa2/ethernet/dpaa2-ethtool.c
@@ -39,15 +39,18 @@
 char dpaa2_ethtool_stats[][ETH_GSTRING_LEN] = {
 	"rx frames",
 	"rx bytes",
-	"rx frames dropped",
-	"rx err frames",
+	/* rx frames filtered/policed */
+	"rx filtered frames",
+	/* rx frames dropped with errors */
+	"rx discarded frames",
 	"rx mcast frames",
 	"rx mcast bytes",
 	"rx bcast frames",
 	"rx bcast bytes",
 	"tx frames",
 	"tx bytes",
-	"tx err frames",
+	/* tx frames dropped with errors */
+	"tx discarded frames",
 };
 
 #define DPAA2_ETH_NUM_STATS	ARRAY_SIZE(dpaa2_ethtool_stats)
